modules/navier_stokes/src/auxkernels/kEpsilonViscosityAux.C

```diff
@@ -24,7 +24,8 @@ kEpsilonViscosityAux::validParams()
   params.addRequiredParam<MooseFunctorName>("u", "The velocity in the x direction.");
   params.addParam<MooseFunctorName>("v", "The velocity in the y direction.");
   params.addParam<MooseFunctorName>("w", "The velocity in the z direction.");
-  params.addRequiredParam<MooseFunctorName>(NS::TKE, "Coupled turbulent kinetic energy.");
+  params.addRequiredParam<MooseFunctorName>("k", "Coupled turbulent kinetic energy.");
```

Comment:
To keep it consistent, please only use variable names in the NS namespace

Reply:
I am deprecating the old name "k" and replacing it with the new NS::TKE (= tke)